examples/create_did.rs

```diff
@@ -3,14 +3,17 @@
 
 use identity::crypto::KeyPair;
 use identity::iota::Client;
+use identity::iota::ClientBuilder;
 use identity::iota::Document;
 use identity::iota::Network;
 use identity::iota::Result;
 use identity::iota::TangleRef;
 
-#[smol_potat::main]
+#[tokio::main]
 async fn main() -> Result<()> {
-  let client: Client = Client::new()?;
+  // Create a new client connected to the Testnet (Chrysalis).
+  // Node-syncing has to be disabled for now.
+  let client: Client = ClientBuilder::new().node_sync_disabled().build().await?;
```

Comment:
There should be a `Client::builder` function. very minor but can avoid an extra import